05-03 22:12:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "USE[*] MASTER

IF DB_ID('PKMNDB') IS NULL
	CREATE DATABASE PKMNDB
	
USE PKMNDB

BEGIN TRAN
	SET NOCOUNT ON

	CREATE TABLE DBO.PKMN
	(
		NAME VARCHAR(11) PRIMARY KEY,                                                     
		[NUMBER] SMALLINT UNIQUE NOT NULL,
		WEIGHT FLOAT(1) NOT NULL,                                             
		CONSTRAINT CH_WEIGHT CHECK(WEIGHT>0)
	) "; expected "UPDATE, {"; SQL statement:
USE master

IF DB_ID('PKMNDB') IS NULL
	CREATE DATABASE PKMNDB
	
USE PKMNDB

BEGIN TRAN
	SET NOCOUNT ON

	CREATE TABLE dbo.Pkmn
	(
		name VARCHAR(11) PRIMARY KEY, -- The longest name is 'Fletchinder' with 11 letters
		[number] SMALLINT UNIQUE NOT NULL,
		weight FLOAT(1) NOT NULL, -- Because some moves are affected by weight
		CONSTRAINT ch_weight CHECK(weight>0)
	) [42001-176]
05-03 22:12:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE DBO.[[*]TYPE]
	(
		NAME VARCHAR(8) PRIMARY KEY                                                      
	) "; expected "identifier"; SQL statement:
CREATE TABLE dbo.[Type]
	(
		name VARCHAR(8) PRIMARY KEY -- The longest type name is 'Fighting' with 8 letters
	) [42001-176]
05-03 22:12:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Schema "DBO" not found; SQL statement:
CREATE TABLE dbo.PkmnType
	(
		pkmnName VARCHAR(11),
		pkmnType VARCHAR(8),
		CONSTRAINT pk_pkmnType PRIMARY KEY(pkmnName, pkmnType),
		CONSTRAINT fk_pkmnName FOREIGN KEY(pkmnName) REFERENCES Pkmn,
		CONSTRAINT fk_pkmnType FOREIGN KEY(pkmnType) REFERENCES [Type]
	) [90079-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.Parser.getSchema(Parser.java:655)
	at org.h2.command.Parser.getSchema(Parser.java:662)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5706)
	at org.h2.command.Parser.parseCreate(Parser.java:4122)
	at org.h2.command.Parser.parsePrepared(Parser.java:351)
	at org.h2.command.Parser.parse(Parser.java:306)
	at org.h2.command.Parser.parse(Parser.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:243)
	at org.h2.engine.Session.prepareLocal(Session.java:442)
	at org.h2.engine.Session.prepareCommand(Session.java:384)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:172)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-03 22:12:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Schema "DBO" not found; SQL statement:
CREATE TABLE dbo.TypeWeakness
	(
		userType VARCHAR(8),
		oponentType VARCHAR(8),
		weakFactor FLOAT(1) NOT NULL,
		CONSTRAINT pk_typeeffect PRIMARY KEY(userType, oponentType),
		CONSTRAINT fk_user FOREIGN KEY(userType) REFERENCES [Type],
		CONSTRAINT fk_oponent FOREIGN KEY(oponentType) REFERENCES [Type],
		CONSTRAINT ch_weakFactor CHECK(weakFactor>=0 )
	) [90079-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.Parser.getSchema(Parser.java:655)
	at org.h2.command.Parser.getSchema(Parser.java:662)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5706)
	at org.h2.command.Parser.parseCreate(Parser.java:4122)
	at org.h2.command.Parser.parsePrepared(Parser.java:351)
	at org.h2.command.Parser.parse(Parser.java:306)
	at org.h2.command.Parser.parse(Parser.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:243)
	at org.h2.engine.Session.prepareLocal(Session.java:442)
	at org.h2.engine.Session.prepareCommand(Session.java:384)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:172)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-03 22:12:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Schema "DBO" not found; SQL statement:
CREATE TABLE dbo.TypeResistance
	(
		userType VARCHAR(8),
		oponentType VARCHAR(8),
		resistanceFactor FLOAT(1) NOT NULL,
		CONSTRAINT pk_typeeffect PRIMARY KEY(userType, oponentType),
		CONSTRAINT fk_user FOREIGN KEY(userType) REFERENCES [Type],
		CONSTRAINT fk_oponent FOREIGN KEY(oponentType) REFERENCES [Type],
		CONSTRAINT ch_resistanceFactor CHECK(resistanceFactor>=0 )
	) [90079-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.Parser.getSchema(Parser.java:655)
	at org.h2.command.Parser.getSchema(Parser.java:662)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5706)
	at org.h2.command.Parser.parseCreate(Parser.java:4122)
	at org.h2.command.Parser.parsePrepared(Parser.java:351)
	at org.h2.command.Parser.parse(Parser.java:306)
	at org.h2.command.Parser.parse(Parser.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:243)
	at org.h2.engine.Session.prepareLocal(Session.java:442)
	at org.h2.engine.Session.prepareCommand(Session.java:384)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:172)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-03 22:12:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "BEGIN TRAN[*]
	SET NOCOUNT ON

	CREATE TABLE DBO.PKMN
	(
		NAME VARCHAR(11) PRIMARY KEY,                                                     
		[NUMBER] SMALLINT UNIQUE NOT NULL,
		WEIGHT FLOAT(1) NOT NULL,                                             
		CONSTRAINT CH_WEIGHT CHECK(WEIGHT>0)
	) "; SQL statement:
BEGIN TRAN
	SET NOCOUNT ON

	CREATE TABLE dbo.Pkmn
	(
		name VARCHAR(11) PRIMARY KEY, -- The longest name is 'Fletchinder' with 11 letters
		[number] SMALLINT UNIQUE NOT NULL,
		weight FLOAT(1) NOT NULL, -- Because some moves are affected by weight
		CONSTRAINT ch_weight CHECK(weight>0)
	) [42000-176]
05-03 22:12:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE DBO.[[*]TYPE]
	(
		NAME VARCHAR(8) PRIMARY KEY                                                      
	) "; expected "identifier"; SQL statement:
CREATE TABLE dbo.[Type]
	(
		name VARCHAR(8) PRIMARY KEY -- The longest type name is 'Fighting' with 8 letters
	) [42001-176]
05-03 22:12:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Schema "DBO" not found; SQL statement:
CREATE TABLE dbo.PkmnType
	(
		pkmnName VARCHAR(11),
		pkmnType VARCHAR(8),
		CONSTRAINT pk_pkmnType PRIMARY KEY(pkmnName, pkmnType),
		CONSTRAINT fk_pkmnName FOREIGN KEY(pkmnName) REFERENCES Pkmn,
		CONSTRAINT fk_pkmnType FOREIGN KEY(pkmnType) REFERENCES [Type]
	) [90079-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.Parser.getSchema(Parser.java:655)
	at org.h2.command.Parser.getSchema(Parser.java:662)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5706)
	at org.h2.command.Parser.parseCreate(Parser.java:4122)
	at org.h2.command.Parser.parsePrepared(Parser.java:351)
	at org.h2.command.Parser.parse(Parser.java:306)
	at org.h2.command.Parser.parse(Parser.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:243)
	at org.h2.engine.Session.prepareLocal(Session.java:442)
	at org.h2.engine.Session.prepareCommand(Session.java:384)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:172)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-03 22:12:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Schema "DBO" not found; SQL statement:
CREATE TABLE dbo.TypeWeakness
	(
		userType VARCHAR(8),
		oponentType VARCHAR(8),
		weakFactor FLOAT(1) NOT NULL,
		CONSTRAINT pk_typeeffect PRIMARY KEY(userType, oponentType),
		CONSTRAINT fk_user FOREIGN KEY(userType) REFERENCES [Type],
		CONSTRAINT fk_oponent FOREIGN KEY(oponentType) REFERENCES [Type],
		CONSTRAINT ch_weakFactor CHECK(weakFactor>=0 )
	) [90079-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.Parser.getSchema(Parser.java:655)
	at org.h2.command.Parser.getSchema(Parser.java:662)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5706)
	at org.h2.command.Parser.parseCreate(Parser.java:4122)
	at org.h2.command.Parser.parsePrepared(Parser.java:351)
	at org.h2.command.Parser.parse(Parser.java:306)
	at org.h2.command.Parser.parse(Parser.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:243)
	at org.h2.engine.Session.prepareLocal(Session.java:442)
	at org.h2.engine.Session.prepareCommand(Session.java:384)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:172)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-03 22:12:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Schema "DBO" not found; SQL statement:
CREATE TABLE dbo.TypeResistance
	(
		userType VARCHAR(8),
		oponentType VARCHAR(8),
		resistanceFactor FLOAT(1) NOT NULL,
		CONSTRAINT pk_typeeffect PRIMARY KEY(userType, oponentType),
		CONSTRAINT fk_user FOREIGN KEY(userType) REFERENCES [Type],
		CONSTRAINT fk_oponent FOREIGN KEY(oponentType) REFERENCES [Type],
		CONSTRAINT ch_resistanceFactor CHECK(resistanceFactor>=0 )
	) [90079-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.Parser.getSchema(Parser.java:655)
	at org.h2.command.Parser.getSchema(Parser.java:662)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5706)
	at org.h2.command.Parser.parseCreate(Parser.java:4122)
	at org.h2.command.Parser.parsePrepared(Parser.java:351)
	at org.h2.command.Parser.parse(Parser.java:306)
	at org.h2.command.Parser.parse(Parser.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:243)
	at org.h2.engine.Session.prepareLocal(Session.java:442)
	at org.h2.engine.Session.prepareCommand(Session.java:384)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:172)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-03 22:15:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "BEGIN TRAN[*]
	SET NOCOUNT ON

	CREATE TABLE PKMN
	(
		NAME VARCHAR(11) PRIMARY KEY,                                                     
		[NUMBER] SMALLINT UNIQUE NOT NULL,
		WEIGHT FLOAT(1) NOT NULL,                                             
		CONSTRAINT CH_WEIGHT CHECK(WEIGHT>0)
 INFORMATION_SCHEMA.TABLES 	) "; SQL statement:
BEGIN TRAN
	SET NOCOUNT ON

	CREATE TABLE Pkmn
	(
		name VARCHAR(11) PRIMARY KEY, -- The longest name is 'Fletchinder' with 11 letters
		[number] SMALLINT UNIQUE NOT NULL,
		weight FLOAT(1) NOT NULL, -- Because some moves are affected by weight
		CONSTRAINT ch_weight CHECK(weight>0)
 INFORMATION_SCHEMA.TABLES 	) [42000-176]
05-03 22:15:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Schema "DBO" not found; SQL statement:
CREATE TABLE dbo.PkmnType
	(
		pkmnName VARCHAR(11),
		pkmnType VARCHAR(8),
		CONSTRAINT pk_pkmnType PRIMARY KEY(pkmnName, pkmnType),
		CONSTRAINT fk_pkmnName FOREIGN KEY(pkmnName) REFERENCES Pkmn,
		CONSTRAINT fk_pkmnType FOREIGN KEY(pkmnType) REFERENCES Type
	) [90079-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.Parser.getSchema(Parser.java:655)
	at org.h2.command.Parser.getSchema(Parser.java:662)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5706)
	at org.h2.command.Parser.parseCreate(Parser.java:4122)
	at org.h2.command.Parser.parsePrepared(Parser.java:351)
	at org.h2.command.Parser.parse(Parser.java:306)
	at org.h2.command.Parser.parse(Parser.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:243)
	at org.h2.engine.Session.prepareLocal(Session.java:442)
	at org.h2.engine.Session.prepareCommand(Session.java:384)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:172)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-03 22:15:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE TYPEWEAKNESS
	(
		USERTYPE VARCHAR(8),
		OPONENTTYPE VARCHAR(8),
		WEAKFACTOR FLOAT(1) NOT NULL,
		CONSTRAINT PK_TYPEEFFECT PRIMARY KEY(USERTYPE, OPONENTTYPE),
		CONSTRAINT FK_USER FOREIGN KEY(USERTYPE) REFERENCES [[*]TYPE],
		CONSTRAINT FK_OPONENT FOREIGN KEY(OPONENTTYPE) REFERENCES [TYPE],
		CONSTRAINT CH_WEAKFACTOR CHECK(WEAKFACTOR>=0 )
	) "; expected "identifier"; SQL statement:
CREATE TABLE TypeWeakness
	(
		userType VARCHAR(8),
		oponentType VARCHAR(8),
		weakFactor FLOAT(1) NOT NULL,
		CONSTRAINT pk_typeeffect PRIMARY KEY(userType, oponentType),
		CONSTRAINT fk_user FOREIGN KEY(userType) REFERENCES [Type],
		CONSTRAINT fk_oponent FOREIGN KEY(oponentType) REFERENCES [Type],
		CONSTRAINT ch_weakFactor CHECK(weakFactor>=0 )
	) [42001-176]
05-03 22:15:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE TYPERESISTANCE
	(
		USERTYPE VARCHAR(8),
		OPONENTTYPE VARCHAR(8),
		RESISTANCEFACTOR FLOAT(1) NOT NULL,
		CONSTRAINT PK_TYPEEFFECT PRIMARY KEY(USERTYPE, OPONENTTYPE),
		CONSTRAINT FK_USER FOREIGN KEY(USERTYPE) REFERENCES [[*]TYPE],
		CONSTRAINT FK_OPONENT FOREIGN KEY(OPONENTTYPE) REFERENCES [TYPE],
		CONSTRAINT CH_RESISTANCEFACTOR CHECK(RESISTANCEFACTOR>=0 )
	) "; expected "identifier"; SQL statement:
CREATE TABLE TypeResistance
	(
		userType VARCHAR(8),
		oponentType VARCHAR(8),
		resistanceFactor FLOAT(1) NOT NULL,
		CONSTRAINT pk_typeeffect PRIMARY KEY(userType, oponentType),
		CONSTRAINT fk_user FOREIGN KEY(userType) REFERENCES [Type],
		CONSTRAINT fk_oponent FOREIGN KEY(oponentType) REFERENCES [Type],
		CONSTRAINT ch_resistanceFactor CHECK(resistanceFactor>=0 )
	) [42001-176]
05-03 22:16:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE PKMN
	(
		NAME VARCHAR(11) PRIMARY KEY,                                                     
		[[*]NUMBER] SMALLINT UNIQUE NOT NULL,
		WEIGHT FLOAT(1) NOT NULL,                                             
		CONSTRAINT CH_WEIGHT CHECK(WEIGHT>0)
	) "; expected "identifier"; SQL statement:
CREATE TABLE Pkmn
	(
		name VARCHAR(11) PRIMARY KEY, -- The longest name is 'Fletchinder' with 11 letters
		[number] SMALLINT UNIQUE NOT NULL,
		weight FLOAT(1) NOT NULL, -- Because some moves are affected by weight
		CONSTRAINT ch_weight CHECK(weight>0)
	) [42001-176]
05-03 22:16:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE PKMN
	(
		NAME VARCHAR(11) PRIMARY KEY,                                                     
		[[*]NUMBER] SMALLINT UNIQUE NOT NULL,
		WEIGHT FLOAT(1) NOT NULL,                                             
		CONSTRAINT CH_WEIGHT CHECK(WEIGHT>0)
	) "; expected "identifier"; SQL statement:
CREATE TABLE Pkmn
	(
		name VARCHAR(11) PRIMARY KEY, -- The longest name is 'Fletchinder' with 11 letters
		[number] SMALLINT UNIQUE NOT NULL,
		weight FLOAT(1) NOT NULL, -- Because some moves are affected by weight
		CONSTRAINT ch_weight CHECK(weight>0)
	) [42001-176]
05-03 22:17:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE PKMN
	(
		NAME VARCHAR(11) PRIMARY KEY,                                                     
		[[*]NUMBER] SMALLINT UNIQUE NOT NULL,
		WEIGHT FLOAT(1) NOT NULL,                                             
		CONSTRAINT CH_WEIGHT CHECK(WEIGHT>0)
	) "; expected "identifier"; SQL statement:
CREATE TABLE Pkmn
	(
		name VARCHAR(11) PRIMARY KEY, -- The longest name is 'Fletchinder' with 11 letters
		[number] SMALLINT UNIQUE NOT NULL,
		weight FLOAT(1) NOT NULL, -- Because some moves are affected by weight
		CONSTRAINT ch_weight CHECK(weight>0)
	) [42001-176]
05-03 22:17:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE PKMN
	(
		NAME VARCHAR(11) PRIMARY KEY,                                                     
		[[*]NUMBER] SMALLINT UNIQUE NOT NULL,
		WEIGHT FLOAT(1) NOT NULL,                                             
		CONSTRAINT CH_WEIGHT CHECK(WEIGHT>0)
	) "; expected "identifier"; SQL statement:
CREATE TABLE Pkmn
	(
		name VARCHAR(11) PRIMARY KEY, -- The longest name is 'Fletchinder' with 11 letters
		[number] SMALLINT UNIQUE NOT NULL,
		weight FLOAT(1) NOT NULL, -- Because some moves are affected by weight
		CONSTRAINT ch_weight CHECK(weight>0)
	) [42001-176]
05-03 22:18:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Schema "DBO" not found; SQL statement:
CREATE TABLE dbo.PkmnType
	(
		pkmnName VARCHAR(11),
		pkmnType VARCHAR(8),
		CONSTRAINT pk_pkmnType PRIMARY KEY(pkmnName, pkmnType),
		CONSTRAINT fk_pkmnName FOREIGN KEY(pkmnName) REFERENCES Pkmn,
		CONSTRAINT fk_pkmnType FOREIGN KEY(pkmnType) REFERENCES Type
	) [90079-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.Parser.getSchema(Parser.java:655)
	at org.h2.command.Parser.getSchema(Parser.java:662)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5706)
	at org.h2.command.Parser.parseCreate(Parser.java:4122)
	at org.h2.command.Parser.parsePrepared(Parser.java:351)
	at org.h2.command.Parser.parse(Parser.java:306)
	at org.h2.command.Parser.parse(Parser.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:243)
	at org.h2.engine.Session.prepareLocal(Session.java:442)
	at org.h2.engine.Session.prepareCommand(Session.java:384)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:172)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-03 22:18:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Schema "DBO" not found; SQL statement:
CREATE TABLE dbo.TypeWeakness
	(
		userType VARCHAR(8),
		oponentType VARCHAR(8),
		weakFactor FLOAT(1) NOT NULL,
		CONSTRAINT pk_typeeffect PRIMARY KEY(userType, oponentType),
		CONSTRAINT fk_user FOREIGN KEY(userType) REFERENCES Type,
		CONSTRAINT fk_oponent FOREIGN KEY(oponentType) REFERENCES Type,
		CONSTRAINT ch_weakFactor CHECK(weakFactor>=0 )
	) [90079-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.Parser.getSchema(Parser.java:655)
	at org.h2.command.Parser.getSchema(Parser.java:662)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5706)
	at org.h2.command.Parser.parseCreate(Parser.java:4122)
	at org.h2.command.Parser.parsePrepared(Parser.java:351)
	at org.h2.command.Parser.parse(Parser.java:306)
	at org.h2.command.Parser.parse(Parser.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:243)
	at org.h2.engine.Session.prepareLocal(Session.java:442)
	at org.h2.engine.Session.prepareCommand(Session.java:384)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:172)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-03 22:18:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Schema "DBO" not found; SQL statement:
CREATE TABLE dbo.TypeResistance
	(
		userType VARCHAR(8),
		oponentType VARCHAR(8),
		resistanceFactor FLOAT(1) NOT NULL,
		CONSTRAINT pk_typeeffect PRIMARY KEY(userType, oponentType),
		CONSTRAINT fk_user FOREIGN KEY(userType) REFERENCES Type,
		CONSTRAINT fk_oponent FOREIGN KEY(oponentType) REFERENCES Type,
		CONSTRAINT ch_resistanceFactor CHECK(resistanceFactor>=0 )
	) [90079-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.Parser.getSchema(Parser.java:655)
	at org.h2.command.Parser.getSchema(Parser.java:662)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5706)
	at org.h2.command.Parser.parseCreate(Parser.java:4122)
	at org.h2.command.Parser.parsePrepared(Parser.java:351)
	at org.h2.command.Parser.parse(Parser.java:306)
	at org.h2.command.Parser.parse(Parser.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:243)
	at org.h2.engine.Session.prepareLocal(Session.java:442)
	at org.h2.engine.Session.prepareCommand(Session.java:384)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:172)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-03 22:18:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Constraint "PK_TYPEEFFECT" already exists; SQL statement:
CREATE TABLE TypeResistance
	(
		userType VARCHAR(8),
		oponentType VARCHAR(8),
		resistanceFactor FLOAT(1) NOT NULL,
		CONSTRAINT pk_typeeffect PRIMARY KEY(userType, oponentType),
		CONSTRAINT fk_user FOREIGN KEY(userType) REFERENCES Type,
		CONSTRAINT fk_oponent FOREIGN KEY(oponentType) REFERENCES Type,
		CONSTRAINT ch_resistanceFactor CHECK(resistanceFactor>=0 )
	) [90045-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:99)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:73)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:170)
	at org.h2.command.CommandContainer.update(CommandContainer.java:79)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-04 12:41:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Constraint "PK_TYPEEFFECT" already exists; SQL statement:
CREATE TABLE TypeResistance
	(
		userType VARCHAR(8),
		oponentType VARCHAR(8),
		resistanceFactor FLOAT(1) NOT NULL,
		CONSTRAINT pk_typeeffect PRIMARY KEY(userType, oponentType),
		CONSTRAINT fk_user FOREIGN KEY(userType) REFERENCES Type,
		CONSTRAINT fk_oponent FOREIGN KEY(oponentType) REFERENCES Type,
		CONSTRAINT ch_resistanceFactor CHECK(resistanceFactor>=0 )
	) [90045-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:99)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:73)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:170)
	at org.h2.command.CommandContainer.update(CommandContainer.java:79)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-04 18:45:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'1,2' (TYPERESISTANCE: RESISTANCEFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPERESISTANCE"("USERTYPE","OPONENTTYPE","RESISTANCEFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 18:45:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'1,2' (TYPERESISTANCE: RESISTANCEFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPERESISTANCE"("USERTYPE","OPONENTTYPE","RESISTANCEFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 18:51:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Constraint "FK_PKMNTYPE" already exists; SQL statement:
CREATE TABLE TypeEffect
	(
		moveType VARCHAR(8),
		pkmnType VARCHAR(8),
		effectivnessFactor FLOAT(1) NOT NULL,
		CONSTRAINT pk_typeEffect PRIMARY KEY(moveType , pkmnType ),
		CONSTRAINT fk_moveType FOREIGN KEY(moveType ) REFERENCES Type,
		CONSTRAINT fk_pkmnType FOREIGN KEY(pkmnType ) REFERENCES Type,
		CONSTRAINT ch_effectivnessFactor  CHECK(effectivnessFactor >=0 )
	) [90045-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:99)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:73)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:170)
	at org.h2.command.CommandContainer.update(CommandContainer.java:79)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-04 18:52:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TYPEEFFECT: EFFECTIVNESSFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPEEFFECT"("MOVETYPE","PKMNTYPE","EFFECTIVNESSFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 18:52:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TYPEEFFECT: EFFECTIVNESSFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPEEFFECT"("MOVETYPE","PKMNTYPE","EFFECTIVNESSFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 18:53:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TYPEEFFECT: EFFECTIVNESSFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPEEFFECT"("MOVETYPE","PKMNTYPE","EFFECTIVNESSFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 18:53:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TYPEEFFECT: EFFECTIVNESSFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPEEFFECT"("MOVETYPE","PKMNTYPE","EFFECTIVNESSFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 18:54:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TYPEEFFECT: EFFECTIVNESSFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPEEFFECT"("MOVETYPE","PKMNTYPE","EFFECTIVNESSFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 18:54:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TYPEEFFECT: EFFECTIVNESSFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPEEFFECT"("MOVETYPE","PKMNTYPE","EFFECTIVNESSFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 18:55:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TYPEEFFECT: EFFECTIVNESSFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPEEFFECT"("MOVETYPE","PKMNTYPE","EFFECTIVNESSFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 18:55:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TYPEEFFECT: EFFECTIVNESSFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPEEFFECT"("MOVETYPE","PKMNTYPE","EFFECTIVNESSFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 19:11:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TYPEEFFECT: EFFECTIVNESSFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPEEFFECT"("MOVETYPE","PKMNTYPE","EFFECTIVNESSFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 19:11:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TYPEEFFECT: EFFECTIVNESSFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPEEFFECT"("MOVETYPE","PKMNTYPE","EFFECTIVNESSFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 19:13:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TYPEEFFECT: EFFECTIVNESSFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPEEFFECT"("MOVETYPE","PKMNTYPE","EFFECTIVNESSFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 19:13:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TYPEEFFECT: EFFECTIVNESSFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPEEFFECT"("MOVETYPE","PKMNTYPE","EFFECTIVNESSFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 19:22:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TYPEEFFECT: EFFECTIVNESSFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPEEFFECT"("MOVETYPE","PKMNTYPE","EFFECTIVNESSFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 19:22:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TYPEEFFECT: EFFECTIVNESSFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPEEFFECT"("MOVETYPE","PKMNTYPE","EFFECTIVNESSFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 19:29:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TYPEEFFECT: EFFECTIVNESSFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPEEFFECT"("MOVETYPE","PKMNTYPE","EFFECTIVNESSFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 19:29:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TYPEEFFECT: EFFECTIVNESSFACTOR FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TYPEEFFECT"("MOVETYPE","PKMNTYPE","EFFECTIVNESSFACTOR")VALUES(?,?,?) -- (?1, ?2, ?3) [22018-176]
05-04 19:46:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Constraint "CH_WEIGHT" not found; SQL statement:
alter table pkmn
drop constraint ch_weight [90057-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.ddl.AlterTableDropConstraint.update(AlterTableDropConstraint.java:42)
	at org.h2.command.CommandContainer.update(CommandContainer.java:79)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
2016-05-04 22:29:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Value too long for column "NAME VARCHAR(11) NOT NULL": "'deoxys-normal' (13)"; SQL statement:
INSERT INTO PKMN(NAME, NUMBER) VALUES(?,?) [22001-191]
05-05 10:33:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "TYPEEFFECT"; expected "TRUNCATE, {"; SQL statement:
TYPEEFFECT [42001-176]
05-05 10:36:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TABLE_NAME" not found; SQL statement:
UPDATE table_name SET MOVETYPE=LOWER(MOVETYPE) [42102-176]
05-05 10:36:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "MOVETYPE" not found; SQL statement:
UPDATE PKMNTYPE SET MOVETYPE=LOWER(MOVETYPE) [42122-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.table.Table.getColumn(Table.java:637)
	at org.h2.command.Parser.readTableColumn(Parser.java:700)
	at org.h2.command.Parser.parseUpdate(Parser.java:734)
	at org.h2.command.Parser.parsePrepared(Parser.java:456)
	at org.h2.command.Parser.parse(Parser.java:306)
	at org.h2.command.Parser.parse(Parser.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:243)
	at org.h2.engine.Session.prepareLocal(Session.java:442)
	at org.h2.engine.Session.prepareCommand(Session.java:384)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:172)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-05 10:47:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
ALTER TABLE PKMN ADD PRIMARY KEY (NUMBER) [90017-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.message.DbException.get(DbException.java:143)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:114)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:73)
	at org.h2.command.CommandContainer.update(CommandContainer.java:79)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-05 10:48:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Schema "SYS" not found; SQL statement:
SELECT * FROM sys.objects

WHERE type_desc LIKE ‘%CONSTRAINT’ [90079-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.Parser.getSchema(Parser.java:655)
	at org.h2.command.Parser.getSchema(Parser.java:662)
	at org.h2.command.Parser.readTableFilter(Parser.java:1186)
	at org.h2.command.Parser.parseSelectSimpleFromPart(Parser.java:1859)
	at org.h2.command.Parser.parseSelectSimple(Parser.java:1968)
	at org.h2.command.Parser.parseSelectSub(Parser.java:1853)
	at org.h2.command.Parser.parseSelectUnion(Parser.java:1674)
	at org.h2.command.Parser.parseSelect(Parser.java:1662)
	at org.h2.command.Parser.parsePrepared(Parser.java:434)
	at org.h2.command.Parser.parse(Parser.java:306)
	at org.h2.command.Parser.parse(Parser.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:243)
	at org.h2.engine.Session.prepareLocal(Session.java:442)
	at org.h2.engine.Session.prepareCommand(Session.java:384)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:172)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-05 10:48:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "OBJECTS" not found; SQL statement:
SELECT * FROM objects

WHERE type_desc LIKE ‘%CONSTRAINT’ [42102-176]
05-05 10:51:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "ALTER TABLE PKMN TYPE[*] DROP CONSTRAINT FK_PKMNNAME_INDEX_8"; expected "., ADD, SET, RENAME, DROP, CHANGE, MODIFY, ALTER"; SQL statement:
ALTER TABLE PKMN TYPE DROP CONSTRAINT FK_PKMNNAME_INDEX_8 [42001-176]
05-05 10:51:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Constraint "FK_PKMNNAME_INDEX_8" not found; SQL statement:
ALTER TABLE PKMNTYPE DROP CONSTRAINT FK_PKMNNAME_INDEX_8 [90057-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.ddl.AlterTableDropConstraint.update(AlterTableDropConstraint.java:42)
	at org.h2.command.CommandContainer.update(CommandContainer.java:79)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-05 10:52:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "ALTER TABLE PKMNTYPE DROP CONSTRAINT FK_PKMNNAME
ALTER[*] TABLE PKMN ADD PRIMARY KEY (NUMBER)"; SQL statement:
ALTER TABLE PKMNTYPE DROP CONSTRAINT FK_PKMNNAME
ALTER TABLE PKMN ADD PRIMARY KEY (NUMBER) [42000-176]
05-05 10:52:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
ALTER TABLE PKMN ADD PRIMARY KEY (NUMBER) [90017-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.message.DbException.get(DbException.java:143)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:114)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:73)
	at org.h2.command.CommandContainer.update(CommandContainer.java:79)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-05 10:54:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Constraint "PRIMARY_KEY" not found; SQL statement:
ALTER TABLE PKMN DROP CONSTRAINT PRIMARY_KEY [90057-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.ddl.AlterTableDropConstraint.update(AlterTableDropConstraint.java:42)
	at org.h2.command.CommandContainer.update(CommandContainer.java:79)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-05 10:55:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "select distinct constraint_name from information_schema.constraints 
where table_name=PKMN' and column_list='INFO[*]'"; SQL statement:
select distinct constraint_name from information_schema.constraints 
where table_name=PKMN' and column_list='INFO' [42000-176]
05-05 10:55:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "select distinct constraint_name from information_schema.constraints 
where table_name=PKMN' and column_list='NAME[*]'"; SQL statement:
select distinct constraint_name from information_schema.constraints 
where table_name=PKMN' and column_list='NAME' [42000-176]
05-05 10:56:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "select distinct constraint_name from information_schema.constraints 
where table_name=PKMN[*]'"; SQL statement:
select distinct constraint_name from information_schema.constraints 
where table_name=PKMN' [42000-176]
05-05 11:01:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column may be referenced by "PUBLIC.PK_PKMNTYPE"; SQL statement:
ALTER TABLE PKMNTYPE
DROP COLUMN PKMNNAME [90083-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.table.Table.dropSingleColumnConstraintsAndIndexes(Table.java:545)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:166)
	at org.h2.command.CommandContainer.update(CommandContainer.java:79)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-05 11:02:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Constraint "PRIMARY_KEY_88" not found; SQL statement:
ALTER TABLE PKMNTYPE DROP CONSTRAINT PRIMARY_KEY_88 [90057-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.ddl.AlterTableDropConstraint.update(AlterTableDropConstraint.java:42)
	at org.h2.command.CommandContainer.update(CommandContainer.java:79)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-05 11:05:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "PKMNID" must not be nullable; SQL statement:
ALTER TABLE PKMNTYPE ADD PRIMARY KEY (PKMNID, PKMNTYPE) [90023-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.table.RegularTable.addIndex(RegularTable.java:206)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:137)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:73)
	at org.h2.command.CommandContainer.update(CommandContainer.java:79)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
2016-05-05 16:26:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "PKMNNAME" not found; SQL statement:
SELECT PKMNTYPE FROM PKMNTYPE WHERE PKMNNAME='dusclops' [42122-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.Comparison.optimize(Comparison.java:178)
	at org.h2.command.dml.Select.prepare(Select.java:855)
	at org.h2.command.Parser.prepareCommand(Parser.java:257)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:73)
	at main.java.pdbi.App.main(App.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144)
05-06 15:53:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DECLARE @[*]V INT
SET @V=1

WHILE @V<=721
BEGIN
UPDATE PKMNTYPE SET PKMNTYPE.PKMNNAME=(SELECT NAME FROM PKMN WHERE NUMBER=@V)
FROM PKMNTYPE WHERE PKMNID=@V
END "; expected "OR, FORCE, VIEW, ALIAS, SEQUENCE, USER, TRIGGER, ROLE, SCHEMA, CONSTANT, DOMAIN, TYPE, DATATYPE, AGGREGATE, LINKED, MEMORY, CACHED, LOCAL, GLOBAL, TEMP, TEMPORARY, TABLE, PRIMARY, UNIQUE, HASH, SPATIAL, INDEX"; SQL statement:
DECLARE @v INT
SET @v=1

WHILE @v<=721
BEGIN
UPDATE PKMNTYPE SET PKMNTYPE.PKMNNAME=(SELECT NAME FROM PKMN WHERE NUMBER=@v)
FROM PKMNTYPE WHERE PKMNID=@v
END [42001-176]
05-06 15:54:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DECLARE @[*]V INT
SET @V=1

WHILE @V<=721
UPDATE PKMNTYPE SET PKMNTYPE.PKMNNAME=(SELECT NAME FROM PKMN WHERE NUMBER=@V)
FROM PKMNTYPE WHERE PKMNID=@V "; expected "OR, FORCE, VIEW, ALIAS, SEQUENCE, USER, TRIGGER, ROLE, SCHEMA, CONSTANT, DOMAIN, TYPE, DATATYPE, AGGREGATE, LINKED, MEMORY, CACHED, LOCAL, GLOBAL, TEMP, TEMPORARY, TABLE, PRIMARY, UNIQUE, HASH, SPATIAL, INDEX"; SQL statement:
DECLARE @v INT
SET @v=1

WHILE @v<=721
UPDATE PKMNTYPE SET PKMNTYPE.PKMNNAME=(SELECT NAME FROM PKMN WHERE NUMBER=@v)
FROM PKMNTYPE WHERE PKMNID=@v [42001-176]
05-06 15:55:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "UPDATE PKMNTYPE SET PKMNTYPE.PKMNNAME=(SELECT NAME FROM PKMN WHERE NUMBER=1)
FROM[*] PKMNTYPE WHERE PKMNID=1"; SQL statement:
UPDATE PKMNTYPE SET PKMNTYPE.PKMNNAME=(SELECT NAME FROM PKMN WHERE NUMBER=1)
FROM PKMNTYPE WHERE PKMNID=1 [42000-176]
2016-05-06 16:03:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "UPDATE PKMNTYPE SET PKMNTYPE.PKMNNAME=(SELECT NAME FROM PKMN WHERE NUMBER=?) FROM[*] PKMNTYPE WHERE PKMNID=?"; SQL statement:
UPDATE PKMNTYPE SET PKMNTYPE.PKMNNAME=(SELECT NAME FROM PKMN WHERE NUMBER=?) FROM PKMNTYPE WHERE PKMNID=? [42000-191]
2016-05-06 16:05:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "UPDATE PKMNTYPE SET PKMNTYPE.PKMNNAME=(SELECT [[*]NAME] FROM PKMN WHERE NUMBER=?) FROM PKMNTYPE WHERE PKMNID=?"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS, SELECT, FROM"; SQL statement:
UPDATE PKMNTYPE SET PKMNTYPE.PKMNNAME=(SELECT [NAME] FROM PKMN WHERE NUMBER=?) FROM PKMNTYPE WHERE PKMNID=? [42001-191]
2016-05-06 16:05:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "UPDATE PKMNTYPE SET PKMNTYPE.PKMNNAME=(SELECT [[*]NAME] FROM PKMN WHERE NUMBER=?) FROM PKMNTYPE WHERE PKMNID=?"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS, SELECT, FROM"; SQL statement:
UPDATE PKMNTYPE SET PKMNTYPE.PKMNNAME=(SELECT [NAME] FROM PKMN WHERE NUMBER=?) FROM PKMNTYPE WHERE PKMNID=? [42001-191]
2016-05-06 16:11:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "UPDATE PKMNTYPE SET PKMNTYPE.PKMNNAME=(SELECT PKMN.NAME FROM PKMN WHERE NUMBER=?) FROM[*] PKMNTYPE WHERE PKMNID=?"; SQL statement:
UPDATE PKMNTYPE SET PKMNTYPE.PKMNNAME=(SELECT PKMN.NAME FROM PKMN WHERE NUMBER=?) FROM PKMNTYPE WHERE PKMNID=? [42000-191]
2016-05-06 16:13:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "UPDATE PKMNTYPE SET PKMNTYPE.PKMNNAME=SELECT NAME FROM PKMN WHERE NUMBER=? FROM[*] PKMNTYPE WHERE PKMNID=?"; SQL statement:
UPDATE PKMNTYPE SET PKMNTYPE.PKMNNAME=SELECT NAME FROM PKMN WHERE NUMBER=? FROM PKMNTYPE WHERE PKMNID=? [42000-191]
05-06 18:05:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Constraint "PRIMARY_KEY" not found; SQL statement:
ALTER TABLE PKMNTYPE DROP CONSTRAINT PRIMARY_KEY [90057-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.command.ddl.AlterTableDropConstraint.update(AlterTableDropConstraint.java:42)
	at org.h2.command.CommandContainer.update(CommandContainer.java:79)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
05-06 18:14:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "PKMNNAME" must not be nullable; SQL statement:
ALTER TABLE PKMNTYPE ADD PRIMARY KEY(PKMNNAME, PKMNTYPE) [90023-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.table.RegularTable.addIndex(RegularTable.java:206)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:137)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:73)
	at org.h2.command.CommandContainer.update(CommandContainer.java:79)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1391)
	at org.h2.server.web.WebApp.query(WebApp.java:1064)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1026)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1013)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
2016-05-06 18:34:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-191]
2016-05-07 23:10:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PKMNEFFECT" not found; SQL statement:
SELECT MOVETYPE, EFFECTIVNESSFACTOR FROM PKMNEFFECT WHERE PKMNTYPE=? OF PKMNTYPE=? [42102-191]
2016-05-07 23:11:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PKMNEFFECT" not found; SQL statement:
SELECT MOVETYPE, EFFECTIVNESSFACTOR FROM PKMNEFFECT WHERE PKMNTYPE=? OF PKMNTYPE=? [42102-191]
2016-05-07 23:12:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PKMNEFFECT" not found; SQL statement:
SELECT MOVETYPE, EFFECTIVNESSFACTOR FROM PKMNEFFECT WHERE PKMNTYPE=? OF PKMNTYPE=? [42102-191]
2016-05-07 23:13:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PKMNEFFECT" not found; SQL statement:
SELECT MOVETYPE, EFFECTIVNESSFACTOR FROM PKMNEFFECT WHERE PKMNTYPE=? OR PKMNTYPE=? [42102-191]
2016-05-07 23:14:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Parameter "#2" is not set; SQL statement:
SELECT MOVETYPE, EFFECTIVNESSFACTOR FROM TYPEEFFECT WHERE PKMNTYPE=? OR PKMNTYPE=? [90012-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.Parameter.checkSet(Parameter.java:81)
	at org.h2.command.Prepared.checkParameters(Prepared.java:164)
	at org.h2.command.CommandContainer.query(CommandContainer.java:109)
	at org.h2.command.Command.executeQuery(Command.java:201)
	at org.h2.jdbc.JdbcPreparedStatement.executeQuery(JdbcPreparedStatement.java:110)
	at pkmntv.dbconn.InfoGetter.typeMapGetter(InfoGetter.java:68)
	at pkmntv.ui.MainWindow.jTextField1ActionPerformed(MainWindow.java:291)
	at pkmntv.ui.MainWindow.access$100(MainWindow.java:17)
	at pkmntv.ui.MainWindow$2.actionPerformed(MainWindow.java:178)
	at javax.swing.JTextField.fireActionPerformed(JTextField.java:508)
	at javax.swing.JTextField.postActionEvent(JTextField.java:721)
	at javax.swing.JTextField$NotifyAction.actionPerformed(JTextField.java:836)
	at javax.swing.SwingUtilities.notifyAction(SwingUtilities.java:1663)
	at javax.swing.JComponent.processKeyBinding(JComponent.java:2880)
	at javax.swing.JComponent.processKeyBindings(JComponent.java:2927)
	at javax.swing.JComponent.processKeyEvent(JComponent.java:2843)
	at java.awt.Component.processEvent(Component.java:6302)
	at java.awt.Container.processEvent(Container.java:2234)
	at java.awt.Component.dispatchEventImpl(Component.java:4881)
	at java.awt.Container.dispatchEventImpl(Container.java:2292)
	at java.awt.Component.dispatchEvent(Component.java:4703)
	at java.awt.KeyboardFocusManager.redispatchEvent(KeyboardFocusManager.java:1954)
	at java.awt.DefaultKeyboardFocusManager.dispatchKeyEvent(DefaultKeyboardFocusManager.java:806)
	at java.awt.DefaultKeyboardFocusManager.preDispatchKeyEvent(DefaultKeyboardFocusManager.java:1074)
	at java.awt.DefaultKeyboardFocusManager.typeAheadAssertions(DefaultKeyboardFocusManager.java:945)
	at java.awt.DefaultKeyboardFocusManager.dispatchEvent(DefaultKeyboardFocusManager.java:771)
	at java.awt.Component.dispatchEventImpl(Component.java:4752)
	at java.awt.Container.dispatchEventImpl(Container.java:2292)
	at java.awt.Window.dispatchEventImpl(Window.java:2739)
	at java.awt.Component.dispatchEvent(Component.java:4703)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:751)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:702)
	at java.awt.EventQueue$3.run(EventQueue.java:696)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.awt.EventQueue$4.run(EventQueue.java:724)
	at java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:721)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
